web/src/EducationBenchmarking.Web/ViewModels/SchoolPlanViewModel.cs

```diff
@@ -0,0 +1,19 @@
+using EducationBenchmarking.Web.Domain;
+
+namespace EducationBenchmarking.Web.ViewModels;
+
+public class SchoolPlanViewModel
```

Comment:
mostly for my own learning here - can this be a primary constructor?

Reply:
Good spot - pushed change